### PR TITLE
Fixes #285

### LINKED ISSUE #285: Dashboard refresh: document the available query string
Update [this page](https://redash.io/help/user-guide/dashboards/dashboard-editing#Dashboard-Refresh) with [this information](https://discuss.redash.io/t/dashboard-that-always-autorefreshes/4432/2).

### PR DESCRIPTION
**Pull request by susodapop**
+ Tightened grammar for this section
+ Added info about the refresh query string variable

### CODE REVIEW

**Comment by susodapop** [comment]
This is related to [#2688](https://github.com/getredash/redash/issues/2688) in the redash repo.

File: src/pages/kb/user-guide/dashboards/dashboard-editing.md [hunk 1]
```diff
@@ -77,4 +79,4 @@ Automatic Dashboard Refresh occurs as part of the Redash frontend application. Y
 
 {% endcallout %}
 
-Importantly, when visitors access a public dashboard they will always see the cached query result. Refreshing a public dashboard does **not** execute the underlying queries. The only way that visitors to a public dashboard link will see refreshed data is if a logged-in Redash user executes the underlying queries or if a query schedule is in effect.
+Importantly for users accessing a public dashboard, the Refresh button will **not** execute the underlying queries. It just checks the cache and updates the dashboard if it finds newer results. The only way that visitors to a public dashboard will see refreshed data is if a logged-in Redash user executes the underlying queries or if a query schedule is in effect.
```
Comment (arikfr) [inline src/pages/kb/user-guide/dashboards/dashboard-editing.md]: @rauchy is this still correct?

Comment (rauchy) [reply src/pages/kb/user-guide/dashboards/dashboard-editing.md]: Nope, public dashboards now execute queries when new parameter values are applied (assuming they are safe).

Comment (susodapop) [reply src/pages/kb/user-guide/dashboards/dashboard-editing.md]: I will update this then. I've restored the branch.

**Pull request merged**